### ISSUE TITLE
Scanning not showing any results. Circles Showing blue.

### ISSUE THREAD

**Issue by HusbandoMax** [AHAAAAAAA/PokemonGo-Map#2837]
https://postimg.org/image/6tv5azy8x/
It had been working fine. Console is showing that its picking up pokemon, gyms and pokestops.

**Comment by NickyBriones** [comment]
SAME PROBLEM ^^^^

**Comment by wearenightowls** [comment]
Niantic shut us down..don't expect this to work anytime soon

**Comment by laernumoretz** [comment]
Sd 5 apparently isn't enough anymore, constant 0s , joy.
This program and similar basically MADE Pokemon go last as long as it did with nearby sucking major ***. 
Numbers will plummet.
Good job niantec, brainless

**Comment by NickyBriones** [comment]
stop spreading bs.... the older versions still work....  http://puu.sh/qkJIR/dd7405aa13.jpg

**Comment by laernumoretz** [comment]
Oh yes, 50+ ppl within an hour decided to all spread the exact same bs.....
Also, your map looks really dry, ever think about that? My old version can pull up abut 4-5 too, which I can do simply pulling my app up and walking for two minutes....

**Comment by NickyBriones** [comment]
sorry I just started the app.. http://puu.sh/qkJUC/7668e820b6.jpg

**Comment by wearenightowls** [comment]
Please do share...also curious to know how an older version can work vs the current repo

**Comment by Pymous** [comment]
Probably because of the multi-threading :)

**Comment by laernumoretz** [comment]
Hmm that's really odd, I've been running the old version pre threading for two days off my pc, it wasn't til about two hours ago that it only started to find about 3 Pokemon off my hidden listen a cycle.
Always 1 thread since it was before threading, st 11-12 no issue, didn't use sd before tonight and was all good, now with sd added I get zeros a lot which sounds backwards,

Something not adding up here if some users old versions are ok but not others (local)
Server issues?

**Comment by Pymous** [comment]
When you use an older version, or set -t to 1, you can send the request, but often, you will just end up with 0 pokemon at the scanned location, that's why almost every map is broken right now.

It's Niantic doing, we will have to wait for the day I think, to let people see how the requests are made In Game now :)

**Comment by wearenightowls** [comment]
![so you re telling me there s a chance](https://cloud.githubusercontent.com/assets/6480247/17276458/ea6b4510-56de-11e6-98c2-e9051be71501.jpg)

**Comment by Pymous** [comment]
Technically, maybe, not my call to make, but it will take time anyway :(

**Comment by NickyBriones** [comment]
So older versions are somewhat ok.... with like 2/3 of what normal spawns are like

**Comment by Pymous** [comment]
2/3 ? You're lucky, I have 3 Pidgeys and 2 Ratatas showing :D

**Comment by externuz** [comment]
We changed `REQ_SLEEP = 6` and `pulse_radius = 0.06`, but it does not helps. So many pokemons... any ideas?

**Comment by Pymous** [comment]
@externuz : Yeah, wait for experienced people to understand what is the limitation Niantic put on the request we can make or not, and hope it's not too much, then wait for an updated version :)

Or use multiple accounts with either multiples IP, or enough delay between the request.

**Comment by externuz** [comment]
We change `REQ_SLEEP` to 5 and `pulse_radius` to 0.03, because nobody can walk more than 30 metres in 5 seconds. But it is so slowly......

**Comment by externuz** [comment]
I think, it should be many workers, imitating real walking for each scan from many accounts. It's only way :-(

**Comment by laernumoretz** [comment]
Exactly externus.  Pretty much what I edited above too, need many accounts

**Comment by externuz** [comment]
I'll see your edit right now. Good idea, but it could be difficult.
P.s. Are you planning some API to your server, to work with others. We use node.js to dynamically create your python servers for each scan and parse STDOUT from each of them to get his state. It would be very usefull to start server once and give him some jobs and have some interfaces to get this jobs state via HTTP/WebSocket/etc. It could be very nice to have possiblity to NOT USE maps and all frontend on your server via some option. Excurse me for my bad English, hope you understand what I mean.

**Comment by A-Metaphysical-Drama** [comment]
I did some research and found some pokemons have a negative "time_till_hidden_ms" parameter so are not showed on the map

I don't know how this works and why they have a negative despawn time but i'm pretty sure these pokemons are showed and catchable in the game

**Comment by pankk** [comment]
@A-Metaphysical-Drama Please see [https://www.reddit.com/r/pokemongodev/comments/4vfwmo/negative_values_for_time_till_hidden_ms/?ref=share&ref_source=link](url). Judging from what ymgve is saying the negative values are actually are correct positive values but parsed incorrectly.
@externuz:

> We change REQ_SLEEP to 5 and pulse_radius to 0.03, because nobody can walk more than 30 metres in 5 seconds. But it is so slowly......

I suppose nobody can walk that quickly, but most of us can surely run at that speed or bike at double that speed... (Disclaimer: I used to be a triathlete before PoGo hit; this game has been really bad for my general health and fitness QQ).
